src/main/java/com/minecolonies/blockout/loader/LoaderManager.java

```diff
@@ -0,0 +1,60 @@
+package com.minecolonies.blockout.loader;
+
+import com.minecolonies.blockout.core.Pane;
+import com.minecolonies.blockout.util.Log;
+import com.minecolonies.blockout.views.View;
+import net.minecraft.util.ResourceLocation;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class LoaderManager
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)